pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,15 +17,15 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repository`
+* `repository` The name of the repository. It is recommended not to include spaces in the repository name.
 * `variants` A multi-valued structure with the following child members:
 +
-** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies.
+** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies. If no value is provided for `name`, this field is defined as *DEFAULT*.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
+`\http://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
 ** `path` The path containing the resource files (for example attributes.adoc).
-** `canonical` Canonical is associated with variants. A module belongs to one or more variants, but there is only one canonical variant.
+** `canonical` Canonical is associated with variants. A module belongs to one or more variants, but there is only one canonical variant. Possible values: *true* or *false*.
```

Comment:
We don't explain *variants* anywhere do we? I think this requires a bit more of an explanation. Also, I still don't understand what a canonical variant is, or what a value or `true` or `false` does functionally in Pv2.